src/uct/ib/dc/base/dc_ep.c

```diff
@@ -32,6 +32,16 @@ static UCS_CLASS_CLEANUP_FUNC(uct_dc_ep_t)
     /* TODO: this is good for dcs policy only.
      * Need to change if eps share dci
      */
+
+    if (!uct_dc_iface_dci_has_outstanding(iface, self->dci)) {
+        ucs_fatal("ifface (%p) ep (%p) dci leak detected: dci=%d",
```

Comment:
Would not be ucs_error be enough here? Even though this is a faulty case, communication can still continue, right?